core/linux-tools/build

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh -uxe
+
+export KBUILD_BUILD_HOST=`hostname`
+export KBUILD_BUILD_USER=`id -nu`
+export CC=gcc
+export CXX=g++
+export BUILD_LD="ld.bfd"
+
+pkgver="$2"
```

Comment:
Please try to avoid naming the script parameters unless you use functions in your script. In this case, just a `sed -i "s|pkgver|2|g"` should fix it